src/redux/categories/categories.js

```diff
@@ -4,12 +4,16 @@ const CHECK_STATUS = 'CHECK_STATUS';
 // Action creators
 export const checkStatus = (payload) => ({ type: CHECK_STATUS, payload });
 
-// Reducer
+// initial state
 const initialState = 'Under construction';
-export default function categoryReducer(state = initialState, action) {
+
+// Reducer
+const categoryReducer = (state = initialState, action) => {
   switch (action.type) {
     case CHECK_STATUS:
       return [...state, action.payload];
     default: return state;
   }
-}
+};
+
+export default categoryReducer;
```

Comment:
- [ ] Good job creating a reducer to check the status but you are missing one more thing. In your React component responsible for categories, add a functionality for "Check status" button click:
  - Import Redux `useDispatch`.
  - Dispatch a corresponding action (import Action Creator that you created before and use it here to display the state on the page).

Reply:
Hi Ricky, @cwaku 

Thanks, the functionality has been added.